CHANGELOG.md

```diff
@@ -1,5 +1,152 @@
 ### [Versions](https://material-ui.com/versions/)
 
+## 5.0.0-alpha.32
+
+<!-- generated comparing v5.0.0-alpha.31..next -->
+
+_Apr 27, 2021_
+
+Big thanks to the 15 contributors who made this release possible. Here are some highlights ✨:
+
+- 👩‍🎤 We finished the migration of all components in `@material-ui/core`. We will focus on the components in `@material-ui/lab` next.
+- 💥 Resume work on the breaking changes, aim for v5.0.0-beta.0 in the next coming months.
+- 📚 Add TypeScript support for adding custom `size` and `color` values in all components.
+- And many more 🐛 bug fixes and 📚 improvements.
+
+### `@material-ui/core@5.0.0-alpha.32`
+
+#### Breaking changes
+
+- <!-- 46 -->  [Table] Rename padding="default" to padding="normal" (#25924) @m4theushw
+
+  ```diff
+  -<Table padding="default" />
+  -<TableCell padding="default" />
+  +<Table padding="normal" />
+  +<TableCell padding="normal" />
+  ```
+
+- <!-- 29 --> [Button] Rename `pending` prop to `loading` in LoadingButton (#25874) @m4theushw
+
+  ```diff
+  -<LoadingButton pending pendingIndicator="Pending..." pendingPosition="end" />
+  +<LoadingButton loading loadingIndicator="Pending..." loadingPosition="end" />
+  ```
+
+- <!-- 25 --> [ButtonBase] Remove buttonRef prop (#25896) @m4theushw
+
+  ```diff
+  -<ButtonBase buttonRef={ref} />
+  +<ButtonBase ref={ref} />
+
+  -<Button buttonRef={ref} />
+  +<Button ref={ref} />
+  ```
```

Comment:
The blank line doesn't show once inside the release page of GitHub

````suggestion
  ```diff
  -<ButtonBase buttonRef={ref} />
  +<ButtonBase ref={ref} />
  ```

  ```diff
  -<Button buttonRef={ref} />
  +<Button ref={ref} />
  ```
````